### ISSUE TITLE
Internationalization of Interface

### ISSUE THREAD

**Issue by ntallman** [psu-libraries/cho-req#163]
As a remote German researcher,
I would like to be able to switch the language of the UI,
So that I can understand how to interact with the content.

Referring to system elements, not content. (Though we could have a translate via Google Translator button for content.)

**Comment by ntallman** [comment]
Related: #52

**Comment by ntallman** [comment]
Related: #93 

We will need to support at least these languages:

> Received from Global Programs in early February 2017:
>
>The top 10 countries represented at Penn State are the following (as of fall 2015):
>
>China- Mandarin/Cantonese (Depending on region. Both should be represented.)
>India- Hindi/Bengali (Depending on region. Both should be represented.)
>S Korea- Korean
>Saudi Arabia- Arabic
>Taiwan- Taiwanese/Hokkien (Depending on region. Both should be represented.)
>Malaysia- Malay: Bahasa Malaysia
>Iran- Persian (Farsi)
>Turkey- Turkish
>United Arab Emirates- Arabic
>Canada- English
>
>Please note: We learned this spring that, even for simple phrases in Arabic, the word order may be >different based on the speaker's country of origin.
